src/js/base/module/LinkDialog.js

```diff
@@ -148,8 +179,10 @@ export default class LinkDialog {
 
       this.ui.onDialogHidden(this.$dialog, () => {
         // detach events
-        $linkText.off();
         $linkUrl.off();
+        $linkText.off();
+        $linkTitle.off();
+        $linkRel.off();
```

Comment:
This do not need off() method.  Because it don't set an event.

Reply:
Ok, I wasn't sure exactly what those were doing, the Text one was already present I think, I can remove all three if you like.

Reply:
`$linkText.off();` was already present before my changes, I just changed the order to reflect the order of the fields in the dialog, I think that's present as there's events attached to update both the Text and URL fields if they are changed.